### PR TITLE
Feat/eip1559

### PR DESCRIPTION
**Pull request by chcmedeiros**

### CODE REVIEW

File: app/src/parser_impl_eth.c [hunk 1]
```diff
@@ -212,59 +243,82 @@ static parser_error_t printERC20(const parser_context_t *ctx, uint8_t displayIdx
...
         case 1:
             snprintf(outKey, outKeyLen, "Contract");
-            rlp_t contractAddress = {.kind = RLP_KIND_STRING, .ptr = eth_tx_obj.legacy.to.ptr, .rlpLen = ETH_ADDRESS_LEN};
+            rlp_t contractAddress = {.kind = RLP_KIND_STRING, .ptr = eth_tx_obj.tx.to.ptr, .rlpLen = ETH_ADDRESS_LEN};
             CHECK_ERROR(printEVMAddress(&contractAddress, outVal, outValLen, pageIdx, pageCount));
             break;
-
         case 2:
+            snprintf(outKey, outKeyLen, "Coin asset");
+            snprintf(outVal, outValLen, "flare");
```
Comment (emmanuelm41) [inline app/src/parser_impl_eth.c:270]: Should this be upper case? Capital letter?

File: app/src/parser_impl_eth.c [hunk 2]
```diff
@@ -284,73 +338,75 @@ static parser_error_t printGeneric(const parser_context_t *ctx, uint8_t displayI
...
     switch (displayIdx) {
         case 0:
             snprintf(outKey, outKeyLen, "To");
-            rlp_t contractAddress = {.kind = RLP_KIND_STRING, .ptr = eth_tx_obj.legacy.to.ptr, .rlpLen = ETH_ADDRESS_LEN};
+            rlp_t contractAddress = {.kind = RLP_KIND_STRING, .ptr = eth_tx_obj.tx.to.ptr, .rlpLen = ETH_ADDRESS_LEN};
             CHECK_ERROR(printEVMAddress(&contractAddress, outVal, outValLen, pageIdx, pageCount));
             break;
         case 1:
+            snprintf(outKey, outKeyLen, "Coin asset");
+            snprintf(outVal, outValLen, "flare");
```
Comment (emmanuelm41) [inline app/src/parser_impl_eth.c:363]: same here

**Review by emmanuelm41 — APPROVED** [review]
Just a simple question.

**Comment by chcmedeiros** [comment]
> @chcmedeiros I don't really have an answer. It was more like a doubt. Are we sure capital letter is ok?

I know. I think the capital letter is ok, and it looks better.

**Pull request merged**